app/src/main/res/layout/activity_departure_list.xml

```diff
@@ -0,0 +1,36 @@
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                                 xmlns:app="http://schemas.android.com/apk/res-auto"
+                                                 xmlns:tools="http://schemas.android.com/tools"
+                                                 android:layout_width="match_parent"
+                                                 android:layout_height="match_parent"
+                                                 android:fitsSystemWindows="true"
+                                                 tools:context=".activity.DepartureListActivity"
+                                                 tools:ignore="MergeRootFrame">
+
+    <android.support.design.widget.AppBarLayout
```

Comment:
This AppBarLayout is the same in all 3 activity_*.xml files. Maybe you could've merged them into one?